public/controllers/agent/register-agent/steps/wz-manager-address.tsx

```diff
@@ -11,14 +13,17 @@ const WzManagerAddressInput = (props: Props) => {
   const [value, setValue] = useState('');
 
   useEffect(() => {
-    if(defaultValue){
+    const prevDefaultValue = usePrevious(defaultValue);
+
+    if (!_.isEqual(defaultValue, prevDefaultValue) && defaultValue !== '') {
       setValue(defaultValue);
       onChange(defaultValue);
-    }else{
+    } else {
       setValue('');
       onChange('');
     }
-  },[])
+  }, []);
```

Comment:
**nitpick**: this is using a non-imported hook. Should we import it or this is not required anymore?

Reply:
Deleted in commit [c0fed1d](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/c0fed1da1dde3abb36fec3b527689b70d48f293f)